static-site/src/components/Groups/styles.jsx

```diff
@@ -35,3 +35,11 @@ export const InputButton = styled.button`
 export const AvatarWithoutMargins = styled(AvatarImg)`
   margin: 0;
 `;
+
+export const TextArea = styled.textarea`
+width: 100%;
+height: 100%;
+font-size: 14px;
+font-family: ${(props) => props.theme.generalFont};
```

Comment:
_non-blocking_

I think a monospace/fixed-width font would be more appropriate/useful here when editing the raw Markdown.

Reply:
Squashed with copy edits and other visual polish in a53b5c7ebb75363142a04d5f745c441f8d21c7c6